test/extended/setup.sh

```diff
@@ -94,6 +94,11 @@ function os::test::extended::setup {
           sudo chcon -t svirt_sandbox_file_t ${VOLUME_DIR}
     fi
     configure_os_server
+    #turn on audit logging for extended tests ... mimic what is done in util.sh configure_os_server, but don't
+    # put change there - only want this for extended tests
+    echo TURNING ON AUDIT LOGGING
```

Comment:
echo "[INFO] Turning on audit logging"
(it's nit picky, but better to align w/ the existing output)